.github/workflows/firebase.yml

```diff
@@ -0,0 +1,64 @@
+name: FireBase
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # run every day at 01:00
+    - cron:  '0 1 * * *'
+
+jobs:
+  assemble_ui_test_artifacts:
+    if: startsWith(github.head_ref, 'master') == true
+    name: Build artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build APK for UI test after Unit tests
+        run: |
+          ./gradlew test
+          ./gradlew assembleDebug
+          ./gradlew assembleDebugAndroidTest
+
+      - name: Upload app-debug APK
+        uses: actions/upload-artifact@v2
+        with:
+          name: app-debug
+          path: app/build/outputs/apk/debug/app-debug.apk
+
+      - name: Upload app-debug-androidTest APK
+        uses: actions/upload-artifact@v2
+        with:
+          name: app-debug-androidTest
+          path: app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk
+
+  run_ui_tests_on_firebase:
+    runs-on: ubuntu-latest
+    needs: assemble_ui_test_artifacts
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download app-debug APK
+        uses: actions/download-artifact@v1
+        with:
+          name: app-debug
+
+      - name: Download app-debug-androidTest APK
+        uses: actions/download-artifact@v1
+        with:
+          name: app-debug-androidTest
+
+      - name: Firebase auth with gcloud
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          service_account_key: ${{ secrets.FIREBASE_KEY }}
+          project_id: ${{ secrets.PROJECT_ID }}
+
+      - name: Run Instrumentation Tests in Firebase Test Lab
+        run: |
+          gcloud firebase test android models list
+          gcloud firebase test android run --type instrumentation --num-flaky-test-attempts 3 --use-orchestrator --app app-debug/app-debug.apk --test app-debug-androidTest/app-debug-androidTest.apk --device model=Pixel2,version=28,locale=en,orientation=portrait
```

Comment:
Is it possible to use strings less then 100 symbols? Because it was a problem for me to find android version:)

Reply:
Done